example-2/evm/src/contracts/lendingHub/HubSetters.sol

```diff
@@ -0,0 +1,156 @@
+
+        setInterestAccrualIndices(assetAddress, accrualIndices);
+
+        // set the max decimals to max of current max and new asset decimals
+        uint8 currentMaxDecimals = getMaxDecimals();
+        if (info.decimals > currentMaxDecimals) {
+            setMaxDecimals(info.decimals);
+        }
+    }
+
+    function consumeMessageHash(bytes32 vmHash) internal {
+        _state.consumedMessages[vmHash] = true;
+    }
+
+    function allowAsset(address assetAddress) internal {
+        _state.allowList.push(assetAddress);
+    }
+
+    function setLastActivityBlockTimestamp(address assetAddress, uint256 blockTimestamp) internal {
+        _state.lastActivityBlockTimestamps[assetAddress] = blockTimestamp;
+    }
+
+    function setInterestAccrualIndices(address assetAddress, AccrualIndices memory indices) internal {
+        _state.indices[assetAddress] = indices;
+    }
+
+    function setInterestAccrualIndexPrecision(uint256 interestAccrualIndexPrecision) internal {
+        _state.interestAccrualIndexPrecision = interestAccrualIndexPrecision;
+    }
+
+    function setCollateralizationRatioPrecision(uint256 collateralizationRatioPrecision) internal {
+        _state.collateralizationRatioPrecision = collateralizationRatioPrecision;
+    }
+
+    function setMaxDecimals(uint8 maxDecimals) internal {
+        _state.MAX_DECIMALS = maxDecimals;
+    }
+
+    function setMaxLiquidationBonus(uint256 maxLiquidationBonus) internal {
+        _state.maxLiquidationBonus = maxLiquidationBonus;
+    }
+
+    function setVaultAmounts(address vaultOwner, address assetAddress, VaultAmount memory vaultAmount) internal {
+        _state.vault[vaultOwner][assetAddress] = vaultAmount;
+    }
+
+    function setGlobalAmounts(address assetAddress, VaultAmount memory vaultAmount) internal {
+        _state.totalAssets[assetAddress] = vaultAmount;
+    }
+
+    function setMaxLiquidationPortion(uint256 maxLiquidationPortion) internal {
+        _state.maxLiquidationPortion = maxLiquidationPortion;
+    }
+
+    function setMaxLiquidationPortionPrecision(uint256 maxLiquidationPortionPrecision) internal {
+        _state.maxLiquidationPortionPrecision = maxLiquidationPortionPrecision;
+    }
+
+    function setMockPyth(uint256 validTimePeriod, uint256 singleUpdateFeeInWei) internal {
+        _state.provider.mockPyth = new MockPyth(validTimePeriod, singleUpdateFeeInWei);
+    }
+
+    function setPriceStandardDeviations(uint64 priceStandardDeviations) internal {
+        _state.priceStandardDeviations = priceStandardDeviations;
+    }
+
+    function setPriceStandardDeviationsPrecision(uint64 priceStandardDeviationsPrecision) internal {
+        _state.priceStandardDeviationsPrecision = priceStandardDeviationsPrecision;
+    }
+
+    function setOraclePrice(bytes32 oracleId, Price memory price) public onlyOwner {
+        _state.oracle[oracleId] = price;
+    }
+
+    function setMockPythFeed(
```

Comment:
It doesn't look like this method is used by anything. Should this be in a test helper?